doc/README.hi.md

```diff
@@ -1,18 +1,19 @@
 <p align="center"><a href="https://newpipe.net"><img src="../assets/new_pipe_icon_5.png" width="150"></a></p> 
-<h2 align="center"><b>NewPipe</b></h2>
+<h2 align="center"><b>न्यूपाइप</b></h2>
 <h4 align="center">Android के लिए एक ओपन सोर्स, हल्का YouTube ऐप।</h4>
 
-<p align="center"><a href="https://f-droid.org/packages/org.schabi.newpipe/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-en.svg" alt="इसे F-Droid पर पाएँ" height=80/></a></p>
+<p align="center"><a href="https://f-droid.org/packages/org.schabi.newpipe/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-hi.svg" alt="इसे F-Droid पर पाएँ" height=80/></a></p>
 
 <p align="center">
 <a href="https://github.com/TeamNewPipe/NewPipe/releases" alt="GitHub प्रकाशन"><img src="https://img.shields.io/github/release/TeamNewPipe/NewPipe.svg" ></a>
 <a href="https://www.gnu.org/licenses/gpl-3.0" alt="लाइसेंस: GPLv3"><img src="https://img.shields.io/badge/License-GPL%20v3-blue.svg"></a>
 <a href="https://github.com/TeamNewPipe/NewPipe/actions" alt="बिल्ड की स्थिति"><img src="https://github.com/TeamNewPipe/NewPipe/workflows/CI/badge.svg?branch=dev&event=push"></a>
 <a href="https://hosted.weblate.org/engage/newpipe/" alt="अनुवाद की स्थिति"><img src="https://hosted.weblate.org/widgets/newpipe/-/svg-badge.svg"></a>
 <a href="https://web.libera.chat/#newpipe" alt="IRC चैनल: #newpipe"><img src="https://img.shields.io/badge/IRC%20chat-%23newpipe-brightgreen.svg"></a>
+<a href="https://matrix.to/#/#newpipe:matrix.newpipe-ev.de" alt="Matrix channel: #newpipe"><img src="https://img.shields.io/badge/Matrix%20chat-%23newpipe-blue"></a>
 </p>
 <hr>
-<p align="center"><a href="#ऐप-कैसी-दिखती-है">ऐप कैसी दिखती है</a> &bull; <a href="#विवरण">विवरण</a> &bull; <a href="#सुविधाएँ">सुविधाएँ</a> &bull; <a href="#स्थापित-करना-और-अपडेट-करना">स्थापित करना और अपडेट करना</a> &bull; <a href="#योगदान-करें">योगदान करें</a> &bull; <a href="#आर्थिक-योगदान-करें">आर्थिक योगदान करें</a> &bull; <a href="#लाइसेंस">लाइसेंस</a></p>
+<p align="center"><a href="#ऐप-कैसी-दिखती-है">ऐप कैसी दिखती है</a> &bull; <a href="#समर्थित-सेवाएँ">समर्थित सेवाएँ</a> &bull; <a href="#विवरण">विवरण</a> &bull; <a href="#सुविधाएँ">सुविधाएँ</a> &bull; <a href="#स्थापित-करना-और-अपडेट-करना">स्थापित करना और अपडेट करना</a> &bull; <a href="#योगदान-करें">योगदान करें</a> &bull; <a href="#आर्थिक-योगदान-करें">आर्थिक योगदान करें</a> &bull; <a href="#लाइसेंस">लाइसेंस</a></p>

```

Comment:
opusforlife, समर्थित सेवाएँ best fits to describe Supported Services here, uplabdh (which means available) does not justify its inclusion because there is a difference between available and supported, as services mentioned in Newpipe README are not only available but also supported continuously. Same goes for "shaamil" which means being present or being added to, while "upasthit" is used for a person when he/she is present on a certain occasion.

Reply:
>The thing is, 'samarthit' is used for the "we're with you"/"we agree with your stance" meaning of 'supported'

Above is the meaning of   "Samarthan",  not "Samrthit". As far as i have seen "Samrthit" is being mostly used as translation of "Supported" in developing community & it is correct too. 

>Personally, 'uplabdh' seems like the best choice to me, primarily because that's how govt services are usually advertised ("yahaan ye sevaaen uplabdh hain").

See English README of NewPipe, word there used specifically is "Supported" which dually covers the meaning of services being "available" and are "Supported" in the application, which describes best the efforts developers make to keep these services running in app,

Maeningwise same goes for being "Samrthit" and being "uplabdh" in hindi. So i would emphasize on retaining word  "समर्थित" .

>don't mark non-'outdated' conversations as resolved until they're finished being discussed

Ok, thanks for  insights on how to use that button during review process, for me it's first time & I am learning.